### PR TITLE
Form for year and sem, redirect pages added

### PR DESCRIPTION
**Pull request by rishabkhincha**

### CODE REVIEW

File: ARC/requirements.txrt [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+et-xmlfile==1.0.1
+jdcal==1.3
+odfpy==1.3.6
+openpyxl==2.5.1
+pytz==2018.3
+PyYAML==3.12
+tablib==0.12.1
+unicodecsv==0.14.1
+xlrd==1.1.0
+xlwt==1.3.0
```
Comment (SebastinSanty) [inline ARC/requirements.txrt]: Extra file?

Comment (rishabkhincha) [reply ARC/requirements.txrt]: Yes, sorry. Deleting.

**Comment by rishabkhincha** [comment]
The code is not complete, @sharanry is on it right now.

File: ARC/main/actions.py [hunk 2]
```diff
@@ -1,55 +1,84 @@
...
+# from main.views import map_options
 
 from django.shortcuts import render
 from django.http import HttpResponse
 from django.template.response import TemplateResponse
-from main.forms import MapsForm
+from main.forms import MapsForm, SingleCDCForm
 
 
 def single_option_CDC(modeladmin, request, queryset):
```
Comment (SebastinSanty) [inline ARC/main/actions.py]: I feel you should be using one convention throughout the code. `forms.py` uses TitleCase and `actions.py` uses case_with_underscore.

**Comment by SebastinSanty** [comment]
@rishabkhincha Yes, just putting it be checked further on.

File: ARC/main/actions.py (hunk 2 above)
Comment (rishabkhincha) [reply ARC/main/actions.py]: Okay, will make those changes

File: ARC/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,14 @@
+diff-match-patch==20121119
```
Comment (sharanry) [inline ARC/requirements.txt:1]: @rishabkhincha Why this extra requirements.txt? There is one already in the root of the repository.

File: ARC/templates/maps/map_options.html [hunk 4]
```diff
@@ -5,7 +5,8 @@
 
 <div id="content-main">
 
-  <form action="" method="POST">
+  <!-- <form action="/mapsuccess" method="POST"> -->
```
Comment (sharanry) [inline ARC/templates/maps/map_options.html:8]: Need to be removed

**Pull request merged**